internal/querynode/query_collection.go

```diff
@@ -47,6 +47,10 @@ import (
 	"github.com/milvus-io/milvus/internal/util/typeutil"
 )
 
+var (
+	defaultLocalCacheSize = 64
```

Comment:
has to be configurable,  and the cache size should be global size of a querynode?

Reply:
and the cache size should be under MB or GB I guess